src/app/feed/feed-card/feed-card.component.html

```diff
@@ -26,6 +26,12 @@
 					[text]="itemText"
 					[config]="cardAutolinkerConfig" (click)="showLightBox.emit({show : 'hide'})"></feed-linker>
 			</div>
+			<div class="card-image" *ngFor="let image of feedItem.images let i = index">
```

Comment:
Sure. As soon as I get some of the changes done , I will let other developers work on this issue more and improve it. :)